src/shared/exit-status.c

```diff
@@ -71,6 +71,7 @@ const ExitStatusMapping exit_status_mappings[256] = {
         [EXIT_CREDENTIALS] =             { "CREDENTIALS",             EXIT_STATUS_SYSTEMD },
         [EXIT_BPF] =                     { "BPF",                     EXIT_STATUS_SYSTEMD },
         [EXIT_KSM] =                     { "KSM",                     EXIT_STATUS_SYSTEMD },
+        [EXIT_THP_DISABLE] =             { "THP_DISABLE",             EXIT_STATUS_SYSTEMD },
```

Comment:
please drop the "disable" word, just call this after the mechanism, hence maybe EXIT_MEMORY_THP or so